pysb/core.py

```diff
@@ -1615,9 +1615,19 @@ def parameters_compartments(self):
         # intersect with original parameter list to retain ordering
         return self.parameters & cset
 
+    def parameters_expressions(self):
+        cset = ComponentSet()
```

Comment:
Add docstring?

Reply:
Docstring added

Reply:
Sorry to nitpick, but can you change it to `Return a ComponentSet of the parameters used in expressions` (similar to `parameters_rules`)? The phrase "expression parameters" is sort of vague.